functions/package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "start": "func host start",
     "test": "jasmine */*.spec.js",
-    "lint": "standard"
+    "lint": "standard",
+    "pack": "funcpack pack ."
```

Comment:
@GuyHarwood - can you confirm this?  You deleted this line yesterday, but it seems to be needed by the Validate containers job.

Reply:
@jon-shipley 
`funcpack` is now deprecated in favour of the new run from package/zip feature.

the lint script should not be removed, that was likely done by accident

Reply:
@GuyHarwood - happy to remove it when the "Validate containers &  MIgrations" job no longer requires it.  See the build failure on PR #864 

Merging is being blocked by this.